packages/commonjs/src/helpers.js

```diff
@@ -172,7 +172,8 @@ export function commonjsResolveImpl (path, originalModuleDir, testCache) {
 	if (path[0] === '/') {
 		originalModuleDir = '/';
 	}
-	while (true) {
+	let depth = 0;
```

Comment:
```suggestion
	var depth = 0;
```

Shouldn’t this be es5?

Reply:
Although actually I already see another `let` and it’s already a template string 🤔

Reply:
It's inside the generated code, not the plugin's code, should be backwards compatible :-)

Reply:
Yeh so rollup itself tries to be compatible with ES3 by default (see https://github.com/rollup/rollup/pull/4215), so I think ideally this should be `var`, and we shouldn't be using template strings